Additional guidance for creating Durable Functions in Azure Portal

gled4er opened the pull request:
I think it will be better if we guide the users to the template. I was a bit confused when I did not see the template in the default template page when Core is pre-selected for Scenario.

azurecla commented:
Hi __@mohitsriv__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mohisri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

gled4er committed:
Additional guidance for creating Durable Function 

I think it will be better if we guide the users to the template. I was a bit confused when I did not see the template in the default template page when Core is pre-selected for Scenario.

cgillum commented on articles/azure-functions/durable-functions-install.md:
```diff
@@ -57,7 +57,7 @@ If you prefer, you can use the Azure portal for Durable Functions development.
 1. Create a new function app at [functions.azure.com](https://functions.azure.com/signin).
 2. Configure the function app to [use the 2.0 runtime version](functions-versions.md).
-3. Create a new function, and select the **Durable Functions Orchestrator - C#** template.
+3. Create a new function, by choosing "C#" for language and "All" for Scenario. Then select the **Durable Functions Orchestrator - C#** template.
```
Shouldn't users select "Durable Functions" for the scenario?

tdykstra replied:
Yes
